build.sbt

```diff
@@ -70,7 +70,7 @@ scalacOptions ++= Seq(
 libraryDependencies ++= Seq(
   "edu.berkeley.cs" %% "chisel3"       % defaultVersions("chisel3"),
   "edu.berkeley.cs" %% "treadle"       % defaultVersions("treadle"),
-  "org.scalatest"   %% "scalatest"     % "3.2.9",
+  "org.scalatest"   %% "scalatest"     % "3.1.2",
```

Comment:
`3.1.4` seems to be the latest `3.1.x` release. Could you try changing it to that?

Reply:
3.1.4 works as well, did do the change